### ISSUE TITLE
Emmet css autocompletion for `d` incorrectly completes to `display: grid` 

### ISSUE THREAD

**Issue by Ciantic** [microsoft/vscode#92120]
- VSCode Version: Win10 64bit 1903
- OS Version: 1.43.0

Steps to Reproduce:

1. Open CSS file
2. Type e.g. 

```
a {
    d // HIT Emmet expand abbrevation
}
```

3. notice now it autocompletes as `display: grid`
```
a {
    display: grid;
}
```

It should however be `display: block` see [Emmet cheat sheet](https://docs.emmet.io/cheat-sheet/) (Someone should make a unit test from that sheet, what do I do with emmet support that is not compliant with the cheat sheet?)

This used to work, and it has been bothering me a few months now. I don't know what changed.

Here is also a screenshot:

![image](https://user-images.githubusercontent.com/64731/76033458-ffaa7f00-5f44-11ea-83c7-4dd4c4f33719.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by yasarsid** [comment]
I can look at this. It seems this was caused by the following [commit](https://github.com/microsoft/vscode-emmet-helper/commit/902d3a0f4ec46629026ed3a727546c62f0fc1f21)

@octref  - If you want I can send out a PR with a fix

**Comment by yasarsid** [comment]
Seemed trivial - raised the PR for your consideration

**Comment by Ciantic** [comment]
Please apply the PR. It seems fine to me!

**Comment by yasarsid** [comment]
@Ciantic  - still awaiting review and merge 👍

**Comment by rzhao271** [comment]
So it should be merged now (as part of 197b9ffeb465d2831b6162da515692525a7d9953) and ready to go in the [insiders](https://code.visualstudio.com/insiders/) build tomorrow.
